.gitignore

```diff
@@ -1 +1,5 @@
 *.pyc
+
+*.swp
+
+tags
```

Comment:
Mind removing the changes to `.gitignore`?

Reply:
*Epic facepalm*

Sorry about that...